guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,254 @@
+
+## Recovery Kit
+Our application goes a bit further than only providing the users with a wallet descriptor. It generates a recovery kit that contains all necessary information in a variety of formats, so that users can easily restore a wallet in case of need. The recovery kit is a ZIP folder that includes:
+
+- **Wallet backup files** compatible with third-party applications like Sparrow or Nunchuk.
+- **A Recovery PDF** with: 
+    - Wallet information and descriptor (as a QR code and in text form)
+    - Details about the two key sets and individual signing keys
+    - Visual representation of the wallet's unlocking rules
+
+If Alice or Bob need to recover the wallet, they can use the Recovery PDF to quickly understand the wallet structure and import it into his preferred wallet software.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   alt-text = "An example of the recovery PDF file."
+   width = 1600
+   height = 800
+   modalWidth = 4050
+   modalHeight = 2168
+%}
+
+## Backup Strategy
+Alice and Bob want to ensure that they can restore the wallet, if they ever need to. They also want to make sure that their children can access the funds if needed, while maintaining security and privacy. They decide to distribute the wallet backup material such that there are two main acces routes to retrieve it. The diagram below provides a high-level overview. 
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   modalImageMobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   alt-text = "A schematic illustration showing how the backup material is distributed between the parents, their children and the lawyer."
+   width = 1600
+   height = 800
+   modalWidth = 3324
+   modalHeight = 1850
+   modalWidthMobile = 1156
+   modalHeightMobile = 2936
+%}
+
+### Primary Access (Self-Recovery)
```

Comment:
I would simplify this and just call it Self-recovery (or similar, primary and secondary don't add much).